gridt/src/app/core/settings.service.ts

```diff
@@ -68,17 +70,22 @@ export class SettingsService {
   private disabler$ = new BehaviorSubject<boolean>(false);
 
   get isDisabled$ (): Observable<boolean> {
-    //debug
-    return of(false)
-    // return this.disabler$.asObservable();
+    return this.disabler$.asObservable();
+  }
+
+  private Dissable(disabler) {
+    return function(error) {
+      disabler.next(true)
+      return throwError(error);
+    };
   }
 
   /**
    * Observable to obtain settings from secure storage
+   * If secure storage is empty, return empty settings object
    */
-  private getLocalSettings$ = this.auth.readyAuthentication$.pipe(
-   flatMap(() => this.secStore.get$("settings")),
-   catchError( this.handleBadAuth(this.disabler$) )
+  private getLocalSettings$: Observable<Settings> = this.auth.readyAuthentication$.pipe(

```

Comment:
Naming convention: this is not a function, so `localSettings$` sounds to me like a more intuitive name for this.